### PR TITLE
Update README - Vets.gov to VA.gov

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This is the bare minimum that we need to do to update our README post-launch. I would like to break this out into a table of contents referencing files in the `docs/` directory later on, but want to get this out there now.

### Ticket
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15144

## Testing done
N/A

## Acceptance criteria
- [x] Readme is accurate and reflects changes that were required during WBC

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# vets.gov - beta [![Build Status](https://dev.vets.gov/jenkins/buildStatus/icon?job=testing/vets-website/master)](http://jenkins.vetsgov-internal/job/department-of-veterans-affairs/job/vets-website/job/master/) [![Test Coverage](https://codeclimate.com/github/department-of-veterans-affairs/vets-website/badges/coverage.svg)](https://codeclimate.com/github/department-of-veterans-affairs/vets-website/coverage)
+# VA.gov [![Build Status](https://dev.vets.gov/jenkins/buildStatus/icon?job=testing/vets-website/master)](http://jenkins.vetsgov-internal/job/department-of-veterans-affairs/job/vets-website/job/master/) [![Test Coverage](https://codeclimate.com/github/department-of-veterans-affairs/vets-website/badges/coverage.svg)](https://codeclimate.com/github/department-of-veterans-affairs/vets-website/coverage)
```
Comment (jbalboni) [inline README.md]: We could remove the code climate stuff.

**Review by jbalboni — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Review by cvalarida — APPROVED** [review]
@erikphansen made some good comments. I have nothing more to add. :stuck_out_tongue:

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ very secret.
 | get the latest json schema               | `yarn remove vets-json-schema; yarn add https://github.com/department-of-veterans-affairs/vets-json-schema.git#{latest commit hash}` |
 | check test coverage                      | `npm run test:coverage`                  |
 | run bundle analyzer on our production JS bundles | `npm run build-analyze`                  |
-| generate a stats file for analysis by bundle analyzer | `NODE_ENV=production npm run build -- -- buildtype production --analyzer`                  |
+| generate a stats file for analysis by bundle analyzer | `NODE_ENV=production npm run build -- -- buildtype=vagovprod --analyzer`                  |
```
[the remark this answers is not in the document]
Comment (ncksllvn) [reply README.md]: We set it in our build script to `production` during `vagovprod` build-types, but I was still a little unsure if we could remove it. From running a local `vagovprod` build now though with `NODE_ENV=production` omitted, it looks like it isn't necessary anymore.

[the remark this answers is not in the document]
Comment (jbalboni) [reply README.md]: You still need to set this. NODE_ENV will put React and other libraries into production mode. It'll work without it, but it's not an accurate reflection of production code without it.

[the remark this answers is not in the document]
Comment (erikphansen) [reply README.md]: ooohhh.... React itself looks for it? That totally makes sense.

[the remark this answers is not in the document]
Comment (jbalboni) [reply README.md]: Hmm, didn't realize that. The staging build is also run in production mode, but it's not set through the build in the same way. I guess we should pick one way to do it?

File: src/site/stages/build/options.js [hunk 3]
```diff
@@ -72,8 +72,16 @@ function applyEnvironmentOverrides(options) {
...
+  const optimizedEnvs = new Set([
+    ENVIRONMENTS.VAGOVPROD,
+    ENVIRONMENTS.VAGOVSTAGING,
+  ]);
+  const isOptimizedBuild = optimizedEnvs.has(options.buildtype);
+
+  if (isOptimizedBuild && process.env.NODE_ENV !== 'production') {
+    throw new Error(
+      'To build a production-like environment, the "NODE_ENV" environment variable must be set to "production".',
+    );
```
Comment (ncksllvn) [inline src/site/stages/build/options.js]: @jbalboni Does this look okay to you?

**Pull request merged**